sql/mozfun/bytes/extract_bits/udf.sql

```diff
@@ -0,0 +1,30 @@
+CREATE OR REPLACE FUNCTION bytes.extract_bits(b BYTES, BEGIN
+  INT64,
+  length INT64
+)
+RETURNS BYTES AS (
```

Comment:
Dear lord, just saw this formatting. We need a way to disable the sql formatter in some cases 😱